gobrew.go

```diff
@@ -671,42 +676,64 @@ func (gb *GoBrew) getGithubTags(repo string) (result []string) {
 
 	githubTags = make(map[string][]string, 0)
 	client := &http.Client{}
-	url := "https://api.github.com/repos/kevincobain2000/gobrew/git/refs/tags"
-	if repo == "golang/go" {
+	url := goTagsApi
+	if repo == golangRepo {
 		url = goBrewTagsApi
 	}
+
 	request, err := http.NewRequest("GET", url, nil)
 	if err != nil {
 		utils.Errorf("[Error] Cannot create request: %s", err)
 		return
 	}
 
-	request.Header.Set("User-Agent", "gobrew")
+	request.Header.Set("User-Agent", gobrewRepo)
 
 	response, err := client.Do(request)
 	if err != nil {
 		utils.Errorf("[Error] Cannot get response: %s", err)
 		return
 	}
-
 	defer func(Body io.ReadCloser) {
 		_ = Body.Close()
 	}(response.Body)
 
-	data, err := io.ReadAll(response.Body)
-	if err != nil {
-		utils.Errorf("[Error] Cannot read response: %s", err)
-		return
-	}
-
 	type Tag struct {
 		Ref string
 	}
 	var tags []Tag
+	var data []byte
+
+	goTagsCacheDir := os.TempDir()
+	cachePath := goTagsCacheDir + strings.Replace(repo, "/", "_", -1) // replace / with _
+
+	// read from local cache
+	// skip error as it may have no cache
+	cachedData, _ := os.ReadFile(cachePath)
```

Comment:
We don't check errors on Readfile. Why?
If we have error on Readfile and we have http.StatusForbidden error its mean we cannot get tags information in this time. And we will not run Unmarshal cacheData, beacouse it not exist.